### DISCUSSION TITLE
Lesson 6

### DISCUSSION THREAD

**Discussion by pragyanthapa** [Cyfrin/foundry-full-course-cu#1608] [Q&A]
<img width="1430" alt="Screenshot 2024-04-16 at 7 46 47 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/155721776/9f27f1ad-e83b-4920-9a8b-18974db6ee78">
Can someone help me why I am not able to install the version of it. I have trying it since hours now

**Comment by EngrPips** [comment]
Hello @pragyanthapa , here is an `ethereum stack exchange post ` https://ethereum.stackexchange.com/questions/149006/foundry-is-not-able-to-install-solc  with a similar issue as yours, maybe go through the post and see if it would help you solve your issue
